src/util/castJsonTypes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Values in the JSON should be appropriately typed however this may not always be the case.
+ * For instance, certain values of a continuous trait may be strings, and we should cast
+ * these to numbers. See https://github.com/nextstrain/auspice/issues/1626 for more discussion
+ * of this particular case. Feel free to add more checks / casts to this function!
+ * @param {Object} metadata
+ * @param {Object} tree
+ * @returns {undefined} Any type casting is in-place
+ */
+export const castIncorrectTypes = (metadata, tree) => {
+  try {
+    const continuousKeys = new Set();
+    Object.entries(metadata.colorings || {}).forEach(([key, details]) => {
+      if (details.type==="continuous") {
+        continuousKeys.add(key);
+      }
+    });
+    tree.nodes.forEach((node) => {
+      Object.entries(node.node_attrs || {}).forEach(([key, details]) => {
+        if (continuousKeys.has(key)) {
+          if ((typeof details.value) !== "number") {
```

Comment:
(_nitpick_) Can these be a single if statement?

Reply:
(New design means this is not applicable)